prestoadmin/server.py

```diff
@@ -82,11 +85,218 @@ def new_sysnode_processor(node_info_rows):
 CONNECTOR_INFO_SQL = 'select catalog_name from system.metadata.catalogs'
 _LOGGER = logging.getLogger(__name__)
 
+DOWNLOAD_DIRECTORY = main_dir
```

Comment:
Agree. It's also not that easy to find since when you install presto-admin, the main_dir becomes buried quite deep.